pre-docs/applying_manifest_blocks.md

```diff
@@ -0,0 +1,189 @@
+# Applying Manifest Blocks
+
+Within a plan, you can use Bolt to apply blocks of Puppet code (manifest blocks) to remote nodes. Similar to the `puppet apply` command - that applies a standalone Puppet manifest to a local system - Bolt leverages manifest blocks to pass code to remote nodes from the command line. You can create manifest blocks that use existing content from the Forge, or mix declarative resource configuration via manifest blocks with procedural orchestration and action in a plan.
+
+**Parent topic:** [Tasks and plans](writing_tasks_and_plans.md)
+
+**Related information**
+
+[Writing plans](writing_plans.md)
+
+## A short example
+
+To get started, we'll show creating a manifest for web server and running it as a plan in Bolt. We'll put this module in Bolt's default Boltdir (`~/.puppetlabs/bolt`).
+
+### nginx on Linux
+
+Setting up a web server using [nginx](https://nginx.org):
+
+1. Go to `~/.puppetlabs/bolt/modules`
+1. Create a new module: `pdk new module profiles` (or create `~/.puppetlabs/bolt/modules/profiles/plans`)
```

Comment:
This command doesn't actually make a `plans` dir, at least using the latest from https://puppet.com/download-puppet-development-kit

Reply:
That's fair. Do you think I need to call out creating the plans dir explicitly? I can drop `/plans` from this step, and the next says create a file in a non-existent directory.

Reply:
Mostly the "or create ~/.puppetlabs/bolt/modules/profiles/plans" made it seem like an exclusive or. I think "Create a new module and add a plans directory: `command` (or create `blee/blah`)" is slightly more clear :woman_shrugging: .

Reply:
Done.